spec.bs

```diff
@@ -468,6 +515,19 @@ try {
     [[webauthn-conditional-ui inline]]. It is likely that both it and SPC could
     be supported by the same underlying API.
 
+: <dfn>Browser bound key</dfn>
+:: A public-private key pair that signs over the transaction in addition to the
+    WebAuthn credential and is tied to a single device by the user agent.
+
+: <dfn>Platform Cryptographic APIs</dfn>
+:: Operating system APIs providing cryptographic algorithms and storage.
+    Particularly browser bound keys require APIs for generating key pairs,
+    retrieving key pairs, exporting public keys, and generating signatures.
+
+: <dfn>Secure element</dfn>
```

Comment:
Similarly, I'm not sure if this is worth/needs to be defined. WebAuthn spec talks about + refers to "Secure Element", "TPM", etc quite freely without defining them (ctrl+f on https://w3c.github.io/webauthn)